common/src/main/java/com/microsoft/identity/common/adal/internal/net/DefaultConnectionService.java

```diff
@@ -42,8 +42,6 @@
 
     private final Context mConnectionContext;
 
-    private static final String TAG = "DefaultConnectionService";
```

Comment:
@iambmelt When are we planning to add logs in common?

Reply:
Because adding logging is a large task, @shoatman and I have prioritized adding relevant logs for ADAL / SSO state sync. This effort has, so far, been centered around classes used in/by `ADALOAuth2TokenCache`.

I can live with other "older" classes using the now `@Deprecated` logging methods for awhile -- they ultimately should transition to the new API before too long. The length of time we're comfortable living with the `@Deprecated` APIs hasn't been determined yet.

So long as the existing logging in this class continues to flow, I think we can postpone the API migration